charts/osm/values.yaml

```diff
@@ -38,24 +38,23 @@ OpenServiceMesh:
   useHTTPSIngress: false
   envoyLogLevel: debug
 
-  # Set deployZipkin to true to deploy a Zipkin cluster in the
-  # namespace where OSM resides. Set this to false if Zipkin
-  # has already been installed or is not needed.
-  deployZipkin: false
+  # Set deployJaeger to true to deploy a Jaeger cluster in the
+  # namespace where OSM resides.
+  deployJaeger: true
 
-  # The following section configures Zipkin tracing for the Envoys
-  # connected to the OSM control plane.
-  zipkin:
```

Comment:
I think it was better to have this key and all the other related keys nested under it. Though I find the naming as zipkin little confusing as it will all be related to the jaeger deployment

Reply:
@shashankram just suggested to have the section renamed to Tracing (intead of zipkin).

The reason for the separation I believe is to separate the "deployment" option with the "endpoint location" config.
would you still believe `deployYagger` should be under `tracingEndpoint` ?